todo_app/data/session_items.py

```diff
@@ -43,27 +43,8 @@ def doneitems(self):
 #loading environment variables for KEY,TOKEN to be used if functions are invoked without flask
 load_dotenv()
 
-#Get id of board created in Trello
 def getBoardid():
-   url_for_board = 'https://api.trello.com/1/members/me/boards'
-
-
-   query = {
-      'key': os.getenv('KEY'),
-      'token': os.getenv('TOKEN')
-   }
-
-   response = requests.request(
-      "GET",
-      url_for_board,
-      params=query
-   )
-   
-   getBoardresponse = response.text
-   getBoardresponse = json.loads(getBoardresponse)
-   return getBoardresponse[0]["id"]
-
-
+    return os.environ['TRELLO_BOARD_ID']
```

Comment:
Minor but note that this changes your app behaviour so we should warn people & make sure the other tests aren't affected;
* Your Readme should mention that this environment variable exists at the same time it warns about the KEY/TOKEN
  * And ideally it should also have those keys (but not values) in the `.env.template` file
* Your `.env.test` needs to have a value for this that matches the value your test expects (`100`) - otherwise the integration test will now fail
  * Also there's a mocked endpoint in there that should no longer be hit that we can get rid of